job/validators.py

```diff
@@ -209,42 +209,36 @@ def cross_clean(self):
         PV tilt set to latitude if not provided and prod_factor_series validated
         """
         def cross_clean_pv(pvmodel):
-            if pvmodel.__getattribute__("tilt") == 0.537:  # 0.537 is a dummy number, default tilt
-                pvmodel.__setattr__("tilt", self.models["Site"].__getattribute__("latitude"))
+            if pvmodel.__getattribute__("tilt") == None:  # 0.537 is a dummy number, default tilt
+                if pvmodel.__getattribute__("array_type") == "ROOFTOP_FIXED":
+                    pvmodel.__setattr__("tilt", 10)
+                else:
+                    pvmodel.__setattr__("tilt", abs(self.models["Site"].__getattribute__("latitude")))
+            
+            if pvmodel.__getattribute__("azimuth") == None:
+                if self.models["Site"].__getattribute__("latitude") >= 0:
+                    pvmodel.__setattr__("azimuth", 180)
+                else:
+                    pvmodel.__setattr__("azimuth", 0.0)
+            
             if pvmodel.__getattribute__("max_kw") > 0:
                 if len(pvmodel.__getattribute__("prod_factor_series")) > 0:
                     self.clean_time_series("PV", "prod_factor_series")
+            
+            if self.models["Settings"].off_grid_flag==True:
+                self.models["PV"].can_net_meter = False
+                self.models["PV"].can_wholesale = False
+                self.models["PV"].can_export_beyond_nem_limit = False
+                self.models["PV"].operating_reserve_required_pct = 0.25
```

Comment:
@rathod-b PV `operating_reserve_required_pct` should not be overwritten if `off_grid_flag=True`. The default should be 0.25 but the user should be allowed to provide another value. Please adjust the validation to reflect this.

Reply:
Addressed in https://github.com/NREL/REopt_API/pull/343/commits/152ac98842eaaf16a7068f0412b34b0893f0886a